packages/bpk-component-infinite-scroll/README.md

```diff
@@ -170,8 +170,8 @@ Updates the internal array and triggers all listeners.
 | loaderMinDisplay        | oneOf(['small', 'half', 'full']) | false    | 'full'        |
 | onScroll                | func                             | false    | null          |
 | onScrollFinished        | func                             | false    | null          |
-| renderLoadingComponent  | func                             | false    | null          |
```

Comment:
This should be changed back to `func` too

Reply:
Good catch! Solved!